tests/ai_hub/model_catalog/plugin_arch/test_readyz_negative.py

```
@@ -0,0 +1,152 @@
+
+        response = poll_readyz(
+            url=readyz_url,
+            headers=model_registry_rest_headers,
+            expected_code=503,
+            timeout=READYZ_UNHEALTHY_TIMEOUT,
+        )
+        body = response.json()
+        assert body["status"] == "not_ready", f"/readyz returned 503 but status is '{body['status']}'"
+        LOGGER.info(f"/readyz returned 503 with body: {body}")
+
+        healthz_response = requests.get(healthz_url, headers=model_registry_rest_headers, verify=False, timeout=10)
```

Ruff (0.15.17)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)